modal/mount.py

```diff
@@ -810,3 +810,110 @@ def _is_modal_path(remote_path: PurePosixPath):
         if is_modal_path:
             return True
     return False
+
+REMOTE_PACKAGES_PATH = "/__modal/deps"
+REMOTE_SITECUSTOMIZE_PATH = "/pkg/sitecustomize.py"
+
+SITECUSTOMIZE_CONTENT = f"""
+# This file is automatically generated by Modal.
+# It ensures that Modal's python dependencies are available in the Python PATH,
+# while prioritizing user-installed packages.
+import sys; sys.path.append('{REMOTE_PACKAGES_PATH}')
+""".strip()
+
+async def _create_single_mount(
```

Comment:
Making these async appears to be necessary for integration tests to work, but I'm not sure why, and could be mistaken. Doesn't hurt, either way I suppose.